### ISSUE TITLE
[Kafka workshop] Workaround for Nuget cert breaking

### ISSUE THREAD

**Issue by SEQUOIIA** [dfds/dojo#15]
Unfortunately we had to put on hold a session of the Kafka workshop last week due to Nuget issues.

TL;DR: A mix of overlapping changes from various vendors caused an issue with verifying certain signed Nuget packages. See the full incident summary at: https://github.com/NuGet/Announcements/issues/49

**Solutions** to continue the previous session:

## 1 [*MUST*]: Make sure to update the Docker images used in the Dockerfiles on your local machine. More namely:

- mcr.microsoft.com/dotnet/sdk:5.0
- mcr.microsoft.com/dotnet/aspnet:5.0

Do a `docker pull` on each of those images before starting the second kata.

## 2 [*PARTLY OPTIONAL*]: Use docker to add Nuget dependency if your OS doesn't trust the newly signed Nuget packages

If you're still encountering the same errors while using your locally installed Dotnet tooling, you can use the updated dotnet Docker image to add a dependency.

Examples of the errors:

_"The author primary signature validity period has expired."_
_"The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain"_

The command you'll be needing will look a bit like this:

```docker run --rm -it --entrypoint sh -v PATH_TO_PROJECT_DIR_LOCALLY:/src mcr.microsoft.com/dotnet/sdk:5.0 -c "cd /src && dotnet add package Confluent.Kafka --version 1.5.3"```

You need to replace *PATH_TO_PROJECT_DIR_LOCALLY* with the **full** path to the project directory of the current kata that you are doing. E.g. for my Linux server where I tested this, it would be something akin to 

_"/home/sequoiia/dfds/dojo/workshops/kafka-deep-dive/2/project"_ 

and on my Windows PC it would be sort of like this 

_"C:\misc\projects\DFDS\ded-dojo\workshops\kafka-deep-dive\2\project"_
